### ISSUE TITLE
Machine Learning Crashes on Windows 10

### ISSUE THREAD

**Issue by osmsc** [SimVascular/SimVascular#559]
**Describe the bug**
Machine Learning no longer works on latest version of SV (2019-07-10) on Windows.  Core error message just before SV crashes:

    error SVWrapper instance not loaded
    Error setting image, sv_wrapper returned null

**To Reproduce**
1. Install SV 2019-07-10
2. Launch SV with DemoProject
3. Select a path (e.g. aorta) and create a new contour group
4. Try and machine learning segment a single slice by clicking the button

**Expected behavior**
A segmentation should appear.  Instead, SimVascular crashes.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - SimVascular Version: 2019-07-10

**Tail of log information to stdout/stderr**

15.17 core.mod.wdgtS.stdMltWdgt: changing layout to 2 x 2D and 3D Widget
16.52 **** Activating legacy standalone functionality
16.52 setting active flag
['C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5\\Lib', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5\\Lib\\site-packages', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\python35.zip', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5\\Lib\\lib-dynload', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5\\Lib\\lib-dynload\\Release', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5\\Lib\\plat-win32', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5\\Lib\\lib-tk', 'C:\\Program Files\\SimVascular\\SimVascular\\2019-07-10\\Python3.5']
3.5.5 (default, Jul  5 2019, 07:04:56) [MSC v.1916 64 bit (AMD64)]
imported svWrapper.py
importing seg_reg modules
importing factories
getting directories
WARNING: Logging before flag parsing goes to stderr.
W0722 16:54:08.136654 27808 deprecation_wrapper.py:119] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\sv_wrapper.py:51: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

SVWrapper init, googlenet_c30_train300k_aug10_clean
C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\config\googlenet_c30_train300k_aug10_clean.yaml
Model dir  C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\results\googlenet_c30_train300k_aug10_clean\model
W0722 16:54:09.308828 27808 lazy_loader.py:50]
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

W0722 16:54:09.312731 27808 deprecation_wrapper.py:119] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\components\models\nn.py:567: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

W0722 16:54:09.314685 27808 deprecation_wrapper.py:119] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\modules\layers.py:424: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

W0722 16:54:09.315661 27808 deprecation_wrapper.py:119] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\modules\layers.py:71: The name tf.random_normal is deprecated. Please use tf.random.normal instead.

pool 1 Tensor("googlenet/max_pool:0", shape=(?, 79, 79, 64), dtype=float32)
pool_2 Tensor("googlenet/max_pool_1:0", shape=(?, 39, 39, 192), dtype=float32)
pool 3 Tensor("googlenet/max_pool_2:0", shape=(?, 19, 19, 480), dtype=float32)
pool side Tensor("googlenet/avg_pool:0", shape=(?, 5, 5, 833), dtype=float32)
[None, 3200]
W0722 16:54:09.682636 27808 deprecation_wrapper.py:119] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\modules\layers.py:108: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

W0722 16:54:09.692396 27808 deprecation.py:506] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\modules\layers.py:484: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
W0722 16:54:09.693371 27808 nn_ops.py:4224] Large dropout rate: 0.7 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.
[None, 1024]
pool 5 Tensor("googlenet/max_pool_3:0", shape=(?, 9, 9, 833), dtype=float32)
pool final Tensor("googlenet/avg_pool_1:0", shape=(?, 3, 3, 1024), dtype=float32)
[None, 9216]
[None, 1024]
Tensor("googlenet/output/Identity:0", shape=(?, 30), dtype=float32)
Tensor("googlenet/side_output/Identity:0", shape=(?, 30), dtype=float32)
W0722 16:54:09.834891 27808 deprecation_wrapper.py:119] From C:\Program Files\SimVascular\SimVascular\2019-07-10\Python3.5\Lib\site-packages\sv_ml\components\models\nn.py:581: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

error SVWrapper instance not loaded
Error setting image, sv_wrapper returned null

**Comment by osmsc** [comment]
@ktbolt & @ gmaher - is the machine learning segmentation working in the latest builds for linux and mac?  See the issue I'm having for Windows.

**Comment by osmsc** [comment]
Fixed in latest release.  Issue was mostly due to an upgrade of Tensorflow version in externals built on 7/5/2019.
